tests/packagedcode/data/windows_helpers/msi/7z1900-x64.msi.expected

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
This is too big, LGPL-licensed. To make things simpler I would consider tow things:
1. have tests that work off the msiinfo output only (eventually refactoring your code along these lines to separate running msitools from parsing the output)
2. have a few minimal tests with tiny .msi files to test the binary parsing proper